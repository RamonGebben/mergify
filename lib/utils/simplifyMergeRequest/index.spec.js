const { path } = require('ramda');
const { simplifyMergeRequest } = require('./index');
const { mrs } = require('./fixtures');
const first = path([0]);

describe('utils/simplifyMergeRequest', () => {
  test('all desired properties are defined', () => {
    const mr = first(mrs);
    expect(simplifyMergeRequest(mr)).toMatchSnapshot();
  });
});
