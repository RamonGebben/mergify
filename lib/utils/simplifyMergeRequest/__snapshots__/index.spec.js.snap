// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils/simplifyMergeRequest all desired properties are defined 1`] = `
Object {
  "assignee": Object {
    "created_at": "2012-04-29T08:46:00Z",
    "email": "admin@example.com",
    "id": 1,
    "name": "Administrator",
    "state": "active",
    "username": "admin",
  },
  "author": Object {
    "created_at": "2012-04-29T08:46:00Z",
    "email": "admin@example.com",
    "id": 1,
    "name": "Administrator",
    "state": "active",
    "username": "admin",
  },
  "id": 1,
  "iid": 1,
  "merge_status": "can_be_merged",
  "source_branch": "test1",
  "state": "opened",
  "target_branch": "master",
  "title": "test1",
  "web_url": "http://example.com/example/example/merge_requests/1",
}
`;
